### ISSUE TITLE
Capture a Keep quest completion

### ISSUE THREAD

**Issue by Ztil** [WarEmu/WarBugs#14829]
**Expected behavior and actual behavior:**
I have been noticing for some time now that the "Capture a Keep" objective never seems to get completed status when capturing it and locking the zone. I searched but couldn't find a bug report on this which is surprising to me. I wonder if it's only happening to me. 

**Steps to reproduce the problem:**
1. Take Capture a Keep quest in currently active RvR zone. 
2. Capture the keep

**Testing Screenshots/Videos/Evidences (always needed):**
Here is a screenshot from Shining Way after locking the Eataine zone and the quest still states Capture a Keep 0/1

![CaptureAKeep](https://user-images.githubusercontent.com/1698328/63616549-0ea09400-c5e8-11e9-8fb4-3a99d9a17e4d.JPG)

**Comment by keunglh** [comment]
Nah, it's not only happening to you. Only the party that gets the killing blow on the lord get credit for it. I think that's existed for as long as I have been playing which is about 9 months, and probably longer.

**Comment by xaraxdestr** [comment]
still an issue, maybe you could deactivate the quests then until it works or change it to an sucessful lock in that zone (since you have to capture keep anyway beside a 5 star lock)
